mx-semantics/main/calls/tools.md

```diff
@@ -83,6 +71,10 @@ module MX-CALLS-TOOLS
                     </mx-call-data> #as MxCallData
                 )
             => pushWorldState
+                // TODO: clearMxReturnValues is not part of the actual MX semantics,
+                // but it makes it easier to identify endpoints that do not
+                // return anything (see setVMOutput-no-retv).
+                ~> clearMxReturnValues
```

Comment:
@virgil-serbanuta could you please elaborate a little bit more on the need to clear the return values? The reasoning is not completely clear to me.

Reply:
Thank you, the TODO text was really bad. Is it better now?

Reply:
It is much clearer now, Virgil. Thank you.